docs/release-notes.asciidoc

```diff
@@ -13,6 +13,24 @@ This section summarizes the changes in each release.
 //[[release-notes-header-7.14]]
 //== 7.14
 
+[discrete]
+[[release-notes-7.14.1]]
+=== 7.14.1
+
+[discrete]
+[[features-7.14.1]]
+==== Features
+The `securitySolution:defaultThreatIndex` advanced setting allows you to specify one or more threat intelligence indices from which {elastic-sec} will collect threat match data. It controls features that query threat indices, such as the Threat Intelligence view on the Overview page, and uses the `filebeat-*` index by default.
+
+You can enable cross cluster search by adding threat indices from remote clusters. Filebeat indices are supported. Non-Filebeat indices require ECS-compliant threat field mappings to ensure threat matching works ({pull}108389[#108389]).
```

Comment:
Same note here as in #993 ; the `Filebeat indices are supported.` sentence raises more questions than it answers, IMO.